Source/WebCore/platform/graphics/gstreamer/TrackPrivateBaseGStreamer.cpp

```diff
@@ -192,10 +192,10 @@ bool TrackPrivateBaseGStreamer::getLanguageCode(GstTagList* tags, AtomString& va
 {
     String language;
     if (getTag(tags, GST_TAG_LANGUAGE_CODE, language)) {
-        language = String::fromLatin1(gst_tag_get_language_code_iso_639_1(language.utf8().data()));
+        language = AtomString::fromLatin1(gst_tag_get_language_code_iso_639_1(language.utf8().data()));
         GST_DEBUG("Converted track %d's language code to %s.", m_index, language.utf8().data());
         if (language != value) {
-            value = language;
+            value = AtomString { language };
```

Comment:
We would then not change this line of code.

Reply:
Ok